benchmarks/Jumps/Manifest.toml

```diff
@@ -1,8 +1,13 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.8.5"
+julia_version = "1.9.0-rc2"
```

Comment:
The CI is still using v1.8.5 which is what's causing the issue.

Reply:
Thanks! Let me change that.